definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -5383,13 +5362,6 @@
     "UserRole": "citizen",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "12/08/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solAosSubmittedUndefended",
```

Comment:
Just need to keep this original object in "AuthorisationCaseEvent-prod.json".

Reply:
Update :9c04f4857977aa362dc8396328aacd041414b378